paths_cli/compiling/strategies.py

```diff
@@ -0,0 +1,102 @@
+from paths_cli.compiling.core import Builder, Parameter
+from paths_cli.compiling.shooting import shooting_selector_compiler
+from paths_cli.compiling.plugins import (
+    StrategyCompilerPlugin, CategoryPlugin
+)
+from paths_cli.compiling.root_compiler import compiler_for
+
+def _strategy_name(class_name):
+    return f"openpathsampling.strategies.{class_name}"
+
+def _group_parameter(group_name):
+    return Parameter('group', str, default=group_name,
+                     description="the group name for these movers")
+
+# TODO: maybe this moves into shooting once we have the metadata?
+SP_SELECTOR_PARAMETER = Parameter('selector', shooting_selector_compiler,
+                                  default=None)
+
+ENGINE_PARAMETER = Parameter('engine', compiler_for('engine'),
+                             description="the engine for moves of this "
+                             "type")
+
+SHOOTING_GROUP_PARAMETER = _group_parameter('shooting')
+REPEX_GROUP_PARAMETER = _group_parameter('repex')
+MINUS_GROUP_PARAMETER = _group_parameter('minus')
+
+REPLACE_TRUE_PARAMETER = Parameter('replace', bool, default=True)
+REPLACE_FALSE_PARAMETER = Parameter('replace', bool, default=False)
+
+
+ONE_WAY_SHOOTING_STRATEGY_PLUGIN = StrategyCompilerPlugin(
+    builder=Builder(_strategy_name("OneWayShootingStrategy")),
+    parameters=[
+        SP_SELECTOR_PARAMETER,
+        ENGINE_PARAMETER,
+        SHOOTING_GROUP_PARAMETER,
+        Parameter('replace', bool, default=True)
```

Comment:
Shouldn't this use `REPLACE_TRUE_PARAMETER` defined before?
```suggestion
        REPLACE_TRUE_PARAMETER
```